### ISSUE TITLE
When KUBE_URL is provided, it failed to connect the KUBE_URL

### ISSUE THREAD

**Issue by aqan213** [draios/sysdig-kube-watcher#9]
When trying to deploy kubewathcer to kubernetes cluster, If the `KUBE_URL` is set in `kubewatcher.yaml` file, it failed to connect this api with `[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed ` and  `403 Forbidden` errors.

Solution: Use the default cert file `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt` and default token `/var/run/secrets/kubernetes.io/serviceaccount/token`.

**Comment by aqan213** [comment]
Error logs:
```
2019-11-25 07:27:25 - info - Kubewatcher Starting
2019-11-25 07:27:25 - info - SDC_URL = https://us-south.monitoring.cloud.ibm.com
/usr/local/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:852: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
2019-11-25 07:27:25 - info - Reading the Kubernetes API
2019-11-25 07:27:25 - error - [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)
Traceback (most recent call last):
  File "kubewatcher.py", line 66, in <module>
    urlparser_ns.parse(KUBE_URL, '/api/v1/namespaces')
  File "/kube_obj_parser.py", line 363, in parse
    resp = self._kube_get(url, endpoint)
  File "/kube_obj_parser.py", line 408, in _kube_get
    return requests.get(url + endpoint)
  File "/usr/local/lib/python2.7/site-packages/requests/api.py", line 70, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/api.py", line 56, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/sessions.py", line 488, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/sessions.py", line 609, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/requests/adapters.py", line 497, in send
    raise SSLError(e, request=request)
SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)
```

**Comment by davideschiera** [comment]
Fixed via #10 and released with sysdig/kubewatcher:0.3.0.
